packages/elements/src/slider/__test__/slider.event.test.js

```diff
@@ -0,0 +1,1465 @@
+import {
+  fixture,
+  expect,
+  elementUpdated,
+  oneEvent,
+} from '@refinitiv-ui/test-helpers';
+
+import '@refinitiv-ui/elements/slider';
+import '@refinitiv-ui/elemental-theme/light/ef-slider';
+
+import { tabSliderPosition, calculateValue } from './utils';
+
+describe('slider/Events', () => {
+  describe('Events', () => {
+    let el;
+
+    beforeEach(async () => {
+      el = await fixture('<ef-slider></ef-slider>');
+    });
+
+    it('Drag thumb slider on desktop', async () => {
+      await elementUpdated(el);
+      setTimeout(() =>
+        el.sliderRef.value.dispatchEvent(new MouseEvent('mousedown'))
```

Comment:
I can see that you use a lot of `el.sliderRef.value` private accessors. I understand this is unavoidable, but it would be better to create a function, e.g.
```
const getSliderElement = el => el.sliderRef.value;
```
So if the private implementation of the code changes you do not need to change tests. I would apply this pattern for *all* private accessors in tests.

Reply:
Done.